src/System.Drawing.Common/src/System/Drawing/Drawing2D/GraphicsPath.Windows.cs

```diff
@@ -183,46 +183,36 @@ public FillMode FillMode
             }
         }
 
-        private PathData _GetPathData()
+        private unsafe PathData _GetPathData()
         {
-            int ptSize = Marshal.SizeOf(typeof(GPPOINTF));
+            int count = PointCount;
 
-            int numPts = PointCount;
+            PathData pathData = new PathData()
+            {
+                Types = new byte[count],
+                Points = new PointF[count]
+            };
 
-            PathData pathData = new PathData() { Types = new byte[numPts] };
+            if (count == 0)
+                return pathData;
```

Comment:
How common is this case?  If common, consider moving it earlier and doing:
```C#
if (count == 0)
{
    return new PathData() { Types = Array.Empty<byte>(), Points = Array.Empty<PointF>() };
}
```

Reply:
Not common.